docs/Configuration.md

```diff
@@ -1,4 +1,8 @@
-By default, the mainnet bcoin config file will reside in ~/.bcoin/bcoin.conf.
+By default, the mainnet bcoin config files will reside in ~/.bcoin/bcoin.conf and ~/.bcoin/wallet.conf.
+If a `network` parameter is passed to bcoin at startup, it will look for a `bcoin.conf` file in the specific
+network directory (and ignore any `network` parameters in that file). For example:
```

Comment:
Could be simplified to say arguments take precedence... (network is not an exception)